mail_no_user_assign_notification/tests/test_mail_no_user_assign_notification.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+from odoo.tests import common, new_test_user
+
+
+class TestMailNoUserAssignNotification(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = new_test_user(cls.env, login="test_user")
+        cls.icp = cls.env.ref(
+            "mail_no_user_assign_notification.no_user_assign_notification_models"
+        )
+        cls.icp.value = "res.partner"
+
+    def test_purchase_order_write(self):
+        partner = self.env["res.partner"].create({"name": "Test partner"})
+        self.assertNotIn(
+            self.user.partner_id, partner.message_follower_ids.mapped("partner_id")
+        )
+        partner.write({"user_id": self.user.id})
+        self.assertNotIn(
```

Comment:
This still testing something different from the purpose of the module.

Reply:
Ok, changed the test approach.